src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -989,3 +1444,48 @@ function getBulkOperationError(error: { type: string; reason?: string }, type: s
       };
   }
 }
+
+/**
+ * Returns an object with the expected version properties. This facilitates Elasticsearch's Optimistic Concurrency Control.
+ *
+ * @param version Optional version specified by the consumer.
+ * @param document Optional existing document that was obtained in a preflight operation.
+ */
+function getExpectedVersionProperties(version?: string, document?: SavedObjectsRawDoc) {
+  if (version) {
+    return decodeRequestVersion(version);
+  } else if (document) {
+    return {
+      if_seq_no: document._seq_no,
+      if_primary_term: document._primary_term,
+    };
+  }
+  return {};
+}
+
+/**
+ * Returns the string representation of a namespace.
+ */
+function getNamespaceString(namespace?: string) {
+  return namespace ?? 'default';
+}
```

Comment:
NIT: `getNamespaceOrDefault` maybe?

Also, if `default` is used multiple times for some checks (did not look) I would create a const for it.

Reply:
The string representation of the default (undefined) namespace is 'default'. I'll update the jsdoc to better describe this.